Bubble_sort/Bubble_sort.rb

```diff
@@ -0,0 +1,26 @@
+def bubble_sort(a)
+
+    t = 1    
+    while t <= a.length - 1
+  
+      i = 0
+      while i <= a.length - t - 1
+    
+        if a[i] > a[i+1]
```

Comment:
演算子の両サイドには半角スペースを開けるのが慣習です。詰めた書き方は初心者感ありますね。
```suggestion
        if a[i] > a[i + 1]
```

もっとも、現場においてはこういった構文的なところはrubocopというGemにお任せするのが一般的です。人間の目でレビューするまでもない、という感じですね。

Reply:
ありがとうございます！rubocopを導入して修正しました！